pkgs/build-support/rust/default.nix

```diff
@@ -144,10 +145,10 @@ stdenv.mkDerivation (args // {
     runHook postBuild
   '';
 
-  checkPhase = args.checkPhase or ''
+  checkPhase = with builtins; args.checkPhase or ''
```

Comment:
I think we prefer using `lib` here. In general having a let-binding like `let flags = lib.concatStringsSep " " cargoTestFlags; in ` would remove the need for the `with` (only need it once anyway) and make the code less repetitive.